bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/binding/ThingConfigAction.java

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright (c) 2014-2015 openHAB UG (haftungsbeschraenkt) and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.smarthome.core.thing.binding;
+
+import org.eclipse.smarthome.core.thing.Thing;
+
+
+/**
+ * This class defines the action configuration interface of Eclipse SmartHome
+ *
+ * @author Chris Jackson - Initial contribution and API
+ */
+public interface ThingConfigAction {
```

Comment:
As I said in the beginning, I've not tested this part, but the idea what that the ThingHandler simply implements the ThingConfigAction interface.